esp/scm/ws_logaccess.ecm

```diff
@@ -56,24 +57,18 @@ ESPResponse GetLogAccessInfoResponse
     string RemoteLogManagerType;
     string RemoteLogManagerConnectionString;
 };
-/*
-* Provides mechanism to query log entries
-*
-* Caller can query by JobId, component, log event type, or target audience by providing the appropriate
-* enumerated value in the LogCategory field, as well as the targeted value in the SearchByValue field.
-*
-* SearchbyValue is optional if LogCategory == ALL
-*
-* Caller should restrict the query to target a specific time range specified in the Range field.
-* By default, the first 100 log entries encountered are returned. Caller to pagenize using the
-* LogLineStartFrom field (specifies as zero-indexed start index) and the LogLineLimit (specifies the maximum
-* number of log entries to be returned)
-* Caller can specify which log columns should be reported via the Columns field, all available columns returned by default.
-*
-* The report format can be specified via the Format field: JSON|XML|CSV
-*/
-ESPRequest GetLogsRequest
+
+ESPenum LogAccessFilterOperator : int
 {
+    NONE(0, "NONE"),
+    AND(1, "AND"),
+    OR(2, "OR"),
+    NOT(3, "NOT")
```

Comment:
Does NOT always imply "AND NOT"?  "OR NOT" would be less common but could occasionally be useful.  You could consider a separate operator / boolean for NOT, or explicit "AND NOT", "OR NOT".

Reply:
no, it's only there for completness. Not currently supported anyway, but not meant to imply binary operation either.